example123_test.go

```diff
@@ -0,0 +1,31 @@
+//go:build go1.23
+// +build go1.23
+
+package rill_test
+
+import (
+	"fmt"
+	"iter"
+
+	"github.com/destel/rill"
+)
+
+func Example_ToSeq2() {
```

Comment:
Depending on the naming of testable examples tehy'll be shown in the different parts of documentation, and will have different urls.
- Example_ToSeq2 - would be a package level example, with url #example-package-ToSeq2
- ExampleToSeq2 - would be an example for ToSeq2, with url #example-ToSeq2

So here we'll need to rename it to ExampleToSeq2, this way it will match the url you have used in the readme file.

You can see how it works on your machine by installing [pkgsite](https://github.com/golang/pkgsite)

Reply:
Also in this function the `doubleNums` variable probably should be called `squares`. Or `x*x` should be replaced with `2*x`. And there's another copy of this code in the readme

Reply:
Fix it at 01b6ecad319d30d82cc8258ad6fe1e3bba4492f2